src/runtime/runtime.js

```diff
@@ -687,6 +687,11 @@ export class ConfiguredRuntime {
     return this.storage_;
   }
 
+  /** @override */
+  clientConfigManager() {
```

Comment:
Could you add a test for this, just to keep the coverage at 100% for the PR?

Reply:
Added.